test/test_dao_1_1.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, **kwargs):
         UnitsID=96,
         UnitsName='degree celsius',
         UnitsType='Temperature',
-        UnitsAbbreviation='°C'),
+        UnitsAbbreviation=u'°C'),
```

Comment:
If the goal is not not support Python <3.3, which I recommend, this is good. The `u''` is innocuous in `Python >=3.4`.